ocs_ci/framework/conf/ocp_version/ocp-4.15-ga-config.yaml

```diff
@@ -0,0 +1,13 @@
+---
+# Config file for OCP GA 4.15 stable channel
+RUN:
+  client_version: '4.15-ga'
+DEPLOYMENT:
+  ocp_url_template: "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/{version}/{file_name}-{os_type}-{version}.tar.gz"
+  installer_version: "4.15-ga"
+  terraform_version: "1.0.11"
+  ignition_version: "3.2.0"
+ENV_DATA:
+  vm_template: "rhcos-4.15.0-x86_64-vmware.x86_64"
+  acm_hub_channel: release-2.9
+  acm_version: "2.9"
```

Comment:
need to add `submariner_version: "0.17.0"`

Reply:
done